File: DSA/pranavdixit_590014418/Day_8_Question_2.py
def character_count(s, char):
    count = s.count(char)
    print(f"Character '{char}' occurs {count} time(s) in \"{s}\"")

# Example usage
character_count("programming", 'r')
character_count("Hello World", 'l')
character_count("JavaScript", 'a')

File: DSA/Ayush_Mishra_590014844/Day_8_Question_2.c
#include <stdio.h>
#include <string.h>
int main() {
    char str[1000], ch;
    int count = 0;

    printf("Enter a string: ");
    fgets(str, sizeof(str), stdin);

    printf("Enter a character to count: ");
    scanf(" %c", &ch);

    for (int i = 0; str[i]; i++) {
        if (str[i] == ch) count++;
    }

    printf("Character '%c' occurs %d time(s).\n", ch, count);
    return 0;
}
